packages/runtime-common/indexer.ts

```diff
@@ -503,6 +529,126 @@ export class Indexer {
     ];
   }
 
+  async searchPrerendered(
+    realmURL: URL,
+    { filter, sort, page }: Query,
+    loader: Loader,
+    opts: QueryOptions = {},
+  ): Promise<{
+    prerenderedCards: PrerenderedCard[];
+    prerenderedCardCssItems: PrerenderedCardCssItem[];
+    meta: QueryResultsMeta;
+  }> {
+    let { results, meta } = await this._search(
```

Comment:
`searchPrerendered`, in the 1st query, now piggybacks on our existing search functionality (this filters, orders, paginates the records) to get the instances, and then there's a 2nd query that fetches all of the unique css for those instances and adds it to the dataset